lib/new_relic/rack/browser_monitoring.rb

```diff
@@ -65,6 +65,10 @@ def should_instrument?(env, status, headers)
           html?(headers) &&
           !attachment?(headers) &&
           !streaming?(env, headers)
+      rescue StandardError => e
```

Comment:
Could do `rescue => e` here but as I'm also cool with it as-is!

Reply:
Aha! You and I were talking about this one recently and I was trying to figure out where my knowledge of a related best practice was coming from.

It turns out that RuboCop prefers it this way, with a named class. See [Style/RescueStandardError](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/RescueStandardError). For some reason this project has (temporarily?) disabled that cop, but I still heed its advice and would prefer to stick with a named class.